### ISSUE TITLE
Logo on MPM website points to a 404

### ISSUE THREAD

**Issue by Lykos** [ReactiveX/rxjs#7359]
### Describe the bug

If you go to the [NPM site of RxJS](https://www.npmjs.com/package/rxjs), there is a broken image with the alt text "RxJS Logo". It points to https://github.com/reactivex/rxjs/blob/HEAD/docs_app/src/assets/images/logos/Rx_Logo_S.png which doesn't exist.

### Expected behavior

The logo should point to https://github.com/ReactiveX/rxjs/blob/0bd47eab10dec89f245b888f1f26e03cb36d2a78/apps/rxjs.dev/src/assets/images/logos/Rx_Logo_S.png which shows the incredibly cute logo of RxJS.

### Reproduction code

```typescript
Just open https://www.npmjs.com/package/rxjs and you'll see the problem.
```

### Reproduction URL

https://www.npmjs.com/package/rxjs

### Version

7.8.1

### Additional context

This is not an issue of the RxJS code itself but rather with its NPM configuration. If you tell me where this is configured, I can even try to fix it myself.

**Comment by jakovljevic-mladen** [comment]
I don't know how it works.

The logo is now moved to `https://raw.githubusercontent.com/reactivex/rxjs/HEAD/apps/rxjs.dev/src/assets/images/logos/Rx_Logo_S.png`, but current stable version is now way off of the `HEAD` so pointing to `https://raw.githubusercontent.com/reactivex/rxjs/HEAD/docs_app/src/assets/images/logos/Rx_Logo_S.png` doesn't really work.

We should probably make it work to point to `7.x` branch, or at least to the latest stable version of the library.

Because it works with `https://raw.githubusercontent.com/reactivex/rxjs/7.x/docs_app/src/assets/images/logos/Rx_Logo_S.png` or `https://raw.githubusercontent.com/reactivex/rxjs/7.8.1/docs_app/src/assets/images/logos/Rx_Logo_S.png`.

Maybe @benlesh has some option to adjust on NPM?

**Comment by benlesh** [comment]
No, we just need to publish 7.x again with the change. It's low-priority, but fixable with a publish.

**Comment by benlesh** [comment]
Hi @Lykos — thank you for taking the time to open this issue and for sharing your thoughtful feedback with the RxJS project. I’m sorry it took us so long to respond. Given how much time has passed and the lack of recent activity, I’m going to close this for now so the tracker reflects the work we’re likely to act on. If this is still relevant on a current RxJS release, please feel welcome to open a new issue with updated context and a reproduction. Thank you again for your contribution and patience.

— Ben
